upgrade/1.2/Stage_1.md

```diff
@@ -5,7 +5,7 @@
 
 ## Procedure
 
-1. (`ncn-m002#`) Apply a workaround for the boot order:
+1. (`ncn-m001#`) Apply a workaround for the boot order:
 
 ```bash
 /usr/share/doc/csm/scripts/workarounds/boot-order/run.sh
```

Comment:
This is to the 1.2 branch, so it should follow the prompt conventions for 1.2.

````suggestion
1. Apply a workaround for the boot order:

```bash
ncn-m001# /usr/share/doc/csm/scripts/workarounds/boot-order/run.sh
````